src/Whoops/Resources/views/frame_list.html.php

```diff
@@ -7,6 +7,17 @@
         <span class="frame-index"><?php echo (count($frames) - $i - 1) ?></span>
         <span class="frame-class"><?php echo $tpl->escape($frame->getClass() ?: '') ?></span>
         <span class="frame-function"><?php echo $tpl->escape($frame->getFunction() ?: '') ?></span>
+
+        <?php if (class_exists('Symfony\Component\VarDumper\Cloner\VarCloner')):
```

Comment:
We could also just add `$tpl->dumpFrameArgs()` which returns an empty string when no dumper is available